packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```diff
@@ -5,34 +5,20 @@
 %global repo_orgname theforeman
 %global repo_name foreman_ygg_worker
 %global yggdrasil_libexecdir %{_libexecdir}/yggdrasil
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-%global yggdrasil_worker_conf_dir %{_root_sysconfdir}/yggdrasil/workers
+%global yggdrasil_worker_conf_dir %{_sysconfdir}/yggdrasil/workers
 
 %global goipath         github.com/%{repo_orgname}/%{repo_name}
 
-%if 0%{?rhel} > 7 && ! 0%{?fedora}
-%define gobuild(o:) \
-        GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback libtrust_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -compressdwarf=false -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v %{?**};
-%else
-%if ! 0%{?gobuild:1}
-%if 0%{?suse_version}
-%define gobuild(o:) GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now '" -a -v %{?**};
-%else
-%define gobuild(o:) GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '" -a -v %{?**};
-%endif
-%endif
-%endif
-
 Name: foreman_ygg_worker
 Version: 0.3.0
 Summary: Worker service for yggdrasil that can act as pull client for Foreman Remote Execution
-Release: 1%{?dist}
+Release: 2%{?dist}
 License: MIT
 
 Source0: https://github.com/%{repo_orgname}/%{repo_name}/releases/download/v%{version}/%{repo_name}-%{version}.tar.gz
 Url: https://github.com/%{repo_orgname}/%{repo_name}/
 
-# EL7 doesn't define go_arches
+# suse leap 15.5 doesn't define go_arches
 %if ! 0%{?go_arches:1}
 %define go_arches %{ix86} x86_64 %{arm} aarch64 ppc64le
 %endif
```

Comment:
`go_arches` is defined if you have `golang-packaging` installed according to what I read:
```suggestion
```

Reply:
No luck it seems
```
Start: rpmbuild foreman_ygg_worker-0.3.0-2.suse.lp155.src.rpm
error: Building target platforms: x86_64
Building for target x86_64
Architecture is not included: x86_64
```